pkgs/development/libraries/libplacebo/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub
+, meson, ninja, pkgconfig
+, vulkan-headers, vulkan-loader, shaderc, glslang, lcms2
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libplacebo";
+  version = "1.18.0";
+
+  src = fetchFromGitHub {
+    owner = "haasn";
```

Comment:
This is a mirror.
Probably should use https://code.videolan.org/videolan/libplacebo